Tools/jit/_stencils.py

```diff
@@ -157,7 +161,7 @@ def as_c(self, where: str) -> str:
             if value:
                 value += " + "
             value += f"(uintptr_t)&{self.symbol}"
-        if _signed(self.addend):
+        if _signed(self.addend) or not value:
```

Comment:
Just so I understand: This change is because the hole can now be a call to `patch_aarch64_trampoline`, with no symbol, no value (`HoleValue.ZERO`), and an integer addend, right?

Reply:
That's correct.